aequilibrae/project/zone.py

```diff
@@ -23,7 +22,7 @@ def __init__(self, dataset: dict, zoning):
 
     def delete(self):
         """Removes the zone from the database"""
-        conn = database_connection()
+        conn = self._project.connect()
```

Comment:
not a problem with this PR, but I notice here that this db connection isn't closed